scripts/src/public/routes/index.ts

```diff
@@ -63,6 +64,7 @@ export function createRoutes(app: express.Express, pathPrefix?: string) {
 	app.get(prefix("config/"), getConfigHandler);
 
 	app.get(prefix("applications/:app_id/blockchain_version"), getAppBlockchainVersion);
+	app.put(prefix("applications/:app_id/blockchain_version"), setAppBlockchainVersion);
```

Comment:
@Doc999tor This must only be enabled for NON production environments - only add this route if this is not production